web/cobrands/sass/_base.scss

```diff
@@ -182,6 +182,11 @@ a {
   }
 }
 
+.meta-2 a[name] {
```

Comment:
Doh, ta. I've done this and also put it in layout rather than base as it doesn't need to apply to base!